meinberlin/apps/livequestions/models.py

```diff
@@ -78,6 +78,10 @@ class LiveStream(module_models.Item):
     )
 
     def save(self, *args, **kwargs):
-        self.live_stream = transforms.clean_html_field(
-            self.live_stream, 'video-editor')
-        super().save(*args, **kwargs)
+        if self.live_stream and '<iframe' in self.live_stream:
+            self.live_stream = transforms.clean_html_field(
+                self.live_stream, 'video-editor')
+            super().save(*args, **kwargs)
+        elif self.__class__.objects.filter(module=self.module).exists():
+            for live_stream in self.__class__.objects.filter(module=self.module):
+                live_stream.delete()
```

Comment:
I don't quite get what it does. Maybe we can talk tomorrow?

Reply:
I should have added some explanation, sorry .. so yes, lets talk about it!

Reply:
@fuzzylogic2000 Ok, I now remeber what I did :) So: the problem was that if the livestream is deleted from the ck editor in dashboard, the object was still there in the db and thus when asking in template for {% if livestream%}, it was true and the 'Live now' was showing. That is why I thought to manually delete the object when the livestream is saved without content (and thus deleted). This is what the `if self.live_stream` part is for.
The `'<iframe' in self.live_stream` part is bc when deleting the livestream in the ckeditor, you actually have to press delete twice, because when only once, there still is an empty div (you can see that in the lower left of ckeditor frame). So if you only press delete once, the empty div will still be saved as livestream object. 
I know this is not a very pretty solution and it would have been better to tell ckeditor to not accept empty divs, but Philli and me tried and didnt succeed..

Reply:
Cool! Thank you!